x-pack/platform/plugins/shared/automatic_import/public/util/oas.test.ts

```diff
@@ -113,1089 +117,92 @@ describe('oas util', () => {
   });
 
   describe('full test', () => {
-    const testSpec: Oas = new Oas({
```

Comment:
this is not deleted, just [moved](https://github.com/elastic/kibana/pull/219466/files#diff-817aae76a181e89b339dd1262f79425a4a3c90f629917f9ad08be00eb94c1993R10) over to `oas_test_helper.ts` so viewing these tests is cleaner